src/hotspot/cpu/aarch64/sharedRuntime_aarch64.cpp

```diff
@@ -2955,255 +2778,5 @@ void OptoRuntime::generate_exception_blob() {
   _exception_blob =  ExceptionBlob::create(&buffer, oop_maps, SimpleRuntimeFrame::framesize >> 1);
 }
 
-// ---------------------------------------------------------------
-
-class NativeInvokerGenerator : public StubCodeGenerator {
```

Comment:
The code below was moved to universalNativeInvoker_aarch64.cpp (and adapted from there)